content/applications/general/email_communication/faq.rst

```diff
@@ -1,173 +1,234 @@
-  must be activated. Then, go to :menuselection:`Settings --> Technical --> Emails`
-
-.. image:: faq/email-retry-technical.png
-   :align: center
-   :alt: Retry button of an emails
+- Use an external outgoing email server to be independent of Odoo's mail limit (refer to the
+  corresponding :doc:`email documentation
+  </applications/general/email_communication/email_servers>`).
+- Wait until 11 PM (UTC) for the daily limit to reset, and retry sending the email. In
+  :ref:`developer mode <developer-mode>`, go to :menuselection:`Settings app --> Technical menu -->
+  Email --> Emails`, then click the :guilabel:`Retry` button next to an unsent email.
 
 .. warning::
-   The daily limit is global to your database and can rise quickly! By default an internal message,
-   a notification, a note, etc. counts as an email in your daily limit if it notifies someone.
-
-You can mitigate this by receiving your :ref:`notifications in Odoo <discuss_app/notification_preferences>`
-instead of by emails.
+   The daily email limit is comprehensive to the database. By default, any internal message,
+   notification, logged note, etc. counts as an email in the daily limit if it notifies someone via
+   email. This can be mitigated by receiving :ref:`notifications in Odoo
+   <discuss_app/notification_preferences>`, instead of emails.
 
-SMTP Error
+SMTP error
 **********
 
-You can find out why an email wasn't transmitted successfully by reviewing the Simple Mail
-Transport Protocol (SMTP) error messages. SMTP is a protocol to describe the email structure
-and transmit it over the Internet, and the error messages generated by email services are helpful
-tools to diagnose and troubleshoot email problems.
+Simple Mail Transport Protocol (SMTP) error messages explain why an email wasn't transmitted
+successfully. :abbr:`SMTP (Simple Mail Transport Protocol)` is a protocol to describe the email
+structure, and transmits data from messages over the Internet. The error messages generated by email
+services are helpful tools to diagnose and troubleshoot email problems.
 
-No Error
-********
+.. example::
+   This is an example of a 554 SMTP permanent delivery error: `554: delivery error: Sorry, your
+   message to ------@yahoo.com cannot be delivered. This mailbox is disabled (554.30). -
+   mta4471.mail.bf1.yahoo.com
+   --- Below this line is a copy of the message.`
 
-Odoo is not always capable of providing information for the reason it failed. The different
-providers implement a personalized policy of the bounce emails and it is not always possible
-for Odoo to interpret it correctly.
+The debug menu can be used to investigate SMTP sending issues from a database. To access the menu,
+:ref:`developer mode <developer-mode>` must be activated. Once activated, navigate to the
+:menuselection:`Debug Menu` in the top right of the menu bar (the :guilabel:`🪲 (bug)` icon),
```

Comment:
The 🪲 icon is too recent (2020), please use an [Emoji 1.0](https://emojipedia.org/emoji-1.0) for device compatibility 🙂
```suggestion
:menuselection:`Debug Menu` in the top right of the menu bar (the :guilabel:`🐞 (bug)` icon),
```